lib/capybara/poltergeist/inspector.rb

```diff
@@ -22,9 +22,12 @@ def url
       "//localhost:#{port}/"
     end
 
-    def open
+    def open(scheme = nil)
       if browser
-        Process.spawn(browser, url)
+        scheme = 'http' unless scheme == 'https' # scheme could be 'file' or 'about' or something else
```

Comment:
It just seems weird to accept scheme and then ignore it with no warning or error if it's not http or https